src/AppContext.ts

```diff
@@ -233,6 +212,30 @@ export function setupAppContext() {
   };
 }
 
+function instant(
+  orderbook$: Observable<Orderbook>,
+  balances$: Observable<Balances>
+) {
+  // TODO: find a way to wait for at least each  of the observables to emit a value
```

Comment:
Does the current implementation have a bug?

Reply:
Good catch. That comment should be removed.